packages/react/stories/bruk.mdx

```diff
@@ -147,3 +147,21 @@ Dette kan løses ved å f.eks bruke [attribute selector](https://developer.mozil
   …
 </table>
 ```
+
+### Hjelpeklasser
+
+Vi benytter oss av enkelte hjelpeklasser som kan brukes mer generelt på tvers av komponenter.
+
+#### `ds-sr-only`
+Skjuler et element visuelt mens det fortsatt er tilgjengelig for skjermlesere.
+
+#### `ds-readonly-icon`
+Legger til et hengelås-ikon via `::before` til elementet klassen settes på. Brukes blant annet i `Field` på label når input er `readonly`. <span className='ds-readonly-icon'>Eksempel</span>.
+
+#### `ds-focus` / `ds-focus--inset`
+Legger til designsystemets fokusring rundt elementet _når det får fokus med tastaturnavigering_ (`:focus-visible`).
+
+#### `ds-focus--visible` / `ds-focus--visible--inset`
```

Comment:
Veldig nice å få dokumentert alt dette 🤗 
Supert å få med at `ds-focus--visible` (i motsetning til `ds-focus`) legger på fokusringen _alltid_, så den er tenkt å bruke i elementer hvor man selv setter opp `:focus-visible` eller `:focus`. Dog, i CSS modules kan man ikke compose inn i en pseudo-selector, kun inn i rene klassenavn `.min-knapp { composes: ... }` , så sånn sett er kanskje `ds-focus--visible` og `ds-focus--visible--inset` trenger å være med i oversikten? 🤔

Reply:
Støtter å legge til tekst om at `*--visible` gjer at den alltid er der.
Eg tenker å dokumentere begge er fint. Den eine blir tenkt til å bruke direkte, den andre i din eigen css?

Reply:
Ja jeg er litt usikker på hva use-casen er for andre enn oss, annet enn å demonstrere hvordan den ser ut 😅

Reply:
Jeg endret litt på teksten og tenker det er like greit å dokumentere disse klassene